maven-surefire-common/src/test/java/org/apache/maven/plugin/surefire/AbstractSurefireMojoTest.java

```diff
@@ -2183,6 +2183,13 @@ public void setToolchain( Toolchain toolchain ) throws Exception
             toolchainField.setAccessible( true );
```

Comment:
pls substitute these three lines with `setInternalState( this, "toolchain", toolchain );`